frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/SlideInDialog.kt

```diff
@@ -111,7 +111,10 @@ class SlideInDialog(
           reattachCheckbox.isEnabled = false
           reattachCheckbox.isSelected = true
         } else {
-          val existsAndHasAChild = entryByName.orNull?.children?.nonEmpty() ?: false
+          var existsAndHasAChild = false
+          if (entryByName != null) {
+            existsAndHasAChild = entryByName.children.nonEmpty() == true
+          }
```

Comment:
Instead of this I'd use `val existsAndHasAChild = entryByName?.children?.nonEmpty() ?: false` as Kotlin makes null handling much easier than Java